web/src/stream_popover.js

```diff
@@ -14,6 +14,7 @@ import * as dropdown_widget from "./dropdown_widget";
 import * as hash_util from "./hash_util";
 import {$t, $t_html} from "./i18n";
 import * as message_edit from "./message_edit";
+import {current} from "./message_lists";
```

Comment:
This should be imported via `import * as`; we prefer to do essentially all imports of modules that way; it helps with finding how code is used via `git grep` considerably.